src/link.js

```diff
@@ -10,6 +12,25 @@ const copyAssetsIOS = require('./ios/copyAssets');
 
 log.heading = 'rnpm-link';
 
+const makeHook = (dependency, name) => (cb) => {
+  if (!dependency.config.hooks || !dependency.config.hooks[name]) {
+    return cb();
+  }
+
+  const hook = spawn(dependency.config.hooks[name], {
+    stdio: 'inherit',
+    stdin: 'inherit',
+  });
+
+  hook.on('close', function prelink(code) {
+    if (code) {
+      log.error(`Failed to link a "${dependency.name}" module`);
+    }
+
+    cb();
```

Comment:
Why it wouldn't be caught? Can't we replace `async.series(... callback || () => {})` (the second noop) with something that actually exits? Or let's just do that in rnpm (exit and log if err from callback)